MetadataMigration/README.md

```diff
@@ -1,299 +1,50 @@
-# Metadata Migration
+# metadata-migration
 
-When performing a migration of a search cluster the metadata items such as indexes, templates, configuration and processes need to be in place before document data can be moved.  The metadata migration tool provides insight into what can be moved, if there are any issues moving that data to the target cluster and can deploy those changes.  By inspecting and analyzing the metadata issues can be discovered early in the overall migration timeline. 
+## What is this tool?
 
-## Migration Lifecycle
+This tool exposes the underlying Reindex-From-Snapshot (RFS) core library in a executable that will migrate the templates and indices in a specified snapshot of a source cluster to a target cluster.  In brief, it parses the contents of the snapshot to extract the settings/configuration of the templates and indices in the snapshot and then migrates those to the target cluster.  The snapshot can either be local on disk or in S3.  The user can apply allowlists to filter which templates/indices are migrated.  If a template or index of the same name already exists on the target cluster, this tool will not overwrite the existing one on the target.  
```

Comment:
Longer term I don't think it makes sense to reference RFS for metadata migration. 
>This tool exposes the underlying Reindex-From-Snapshot (RFS) core library  

nit: in a executable --> in an executable